addons/io_scene_gltf2/blender/exp/gltf2_blender_generate.py

```diff
@@ -1132,15 +1132,13 @@ def generate_meshes(operator,
 
             indices = internal_primitive['indices']
 
-            componentType = GLTF_COMPONENT_TYPE_UNSIGNED_BYTE
+            componentType = GLTF_COMPONENT_TYPE_UNSIGNED_SHORT
 
             max_index = max(indices)
 
-            if max_index < 256:
-                componentType = GLTF_COMPONENT_TYPE_UNSIGNED_BYTE
-            elif max_index < 65536:
+            if max_index < 65535:
                 componentType = GLTF_COMPONENT_TYPE_UNSIGNED_SHORT
-            elif max_index < 4294967296:
+            elif max_index < 4294967295:
                 componentType = GLTF_COMPONENT_TYPE_UNSIGNED_INT
             else:
                 print_console('ERROR', 'Invalid max_index: ' + str(max_index))
```

Comment:
Could the error message here be more descriptive? Something like:
> A mesh contains too many vertices and needs to be split before export.

Reply:
Done